taker/src/main.rs

```diff
@@ -427,7 +306,7 @@ async fn main() -> Result<()> {
         Err(_) => Environment::Binary,
     };
 
-    let price_feed_network = network.price_feed_network();
+    let bitmex_network = network.bitmex_network();
```

Comment:
🙃 Not worth changing now, but it's easier to review this PR if renaming this method happens in a separate commit.

Reply:
will keep in mind next time!